src/main/java/org/ambraproject/rhino/config/HibernateLoggingInterceptor.java

```diff
@@ -77,22 +97,13 @@ public boolean onSave(Object entity, Serializable id, Object[] state, String[] p
   public void onDelete(Object entity, Serializable id, Object[] state, String[] propertyNames, Type[] types) {
     if (entity instanceof Comment) {
       Comment comment = (Comment) entity;
-      String commentDeletionMessage = String.format("Comment deleted. URI: %s ", comment.getCommentUri());
-      sendMessage("ambra-comment-deleted", commentDeletionMessage);
-    } else if (entity instanceof Flag) {
-      Flag flag = (Flag) entity;
-      String flagDeletionMessage = String.format("Comment unflagged. URI: %s ", flag.getFlaggedComment().getCommentUri());
-      sendMessage("ambra-flag-deleted", flagDeletionMessage);
-    } else if (entity instanceof ArticleCategoryAssignmentFlag) {
-      ArticleCategoryAssignmentFlag flag = (ArticleCategoryAssignmentFlag) entity;
-      String flagDeletionMessage = String.format("Category unflagged. Article DOI: %s , Category: %s",
-          flag.getArticle().getDoi(), flag.getCategory());
-      sendMessage("ambra-category-flag-deleted", flagDeletionMessage);
+      KafkaJsonOutput output = new KafkaJsonOutput(comment.getCommentUri(), KafkaEventType.DELETE);
```

Comment:
What do you think of moving the creation of `KafkaJsonOutput` inside `sendMessage`?